tests/functional/_support/FailedInfo.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Codeception\Extension;
+
+use Codeception\Events;
+use Codeception\Extension;
+use Codeception\Event\FailEvent;
+
+/**
+ * Custom extension to output troubleshooting information for failed functional tests.
+ */
+class FailedInfo extends Extension
+{
+    public static $events = array(
```

Comment:
Should we be using the old style `array()` instead of `[]`?

Reply:
Updated syntax.